src/styles/index.scss

```diff
@@ -1,3 +1,162 @@
+
+$main-color: #060b35;
+$secondary-color: #616070;
+$blue-color: #00acdc;
+$white-color: #fff;
+
+:root {
+  --main-color: #{$main-color};
+  --secondary: #{$secondary-color};
+  --blue-color: #{$blue-color};
+  --white: #{$white-color};
+}
+
 body {
```

Comment:
you can also set your `font-family` here, you won't have to type it over and over again